docker-compose.yml

```diff
@@ -42,6 +42,8 @@ services:
     networks:
       - internal
     restart: unless-stopped
+    logging:
```

Comment:
nice, pgadmin4 could be commented out [by explaining why it is commented out], as it is not working at the moment.

Reply:
I think it can still work in background. i just have removed the logs on it.